app/src/main/java/com/example/rickandmortyapp/ui/MainActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.rickandmortyapp.ui
```

Comment:
I removed all business logic from the MainActivity, it is now only responsible to initialize the Navigation graph that will allow you to change from Fragment to Fragment.

Each fragment should be a screen. So the CharacterFragment is the screen responsible to display the Character list.

There's no actual navigation since there is no other screen right now, so you will need to learn how to proper navigate from one Fragment to another.

Hint: you will have to declare things in the nav_graph_main file.